src/hotspot/share/interpreter/bytecodeTracer.cpp

```diff
@@ -129,10 +129,10 @@ class BytecodePrinter {
       int bci = (int)(bcp - method->code_base());
       st->print("[%zu] ", Thread::current()->osthread()->thread_id_for_printing());
       if (Verbose) {
-        st->print("%8d  %4d  " INTPTR_FORMAT " " INTPTR_FORMAT " %s",
+        st->print("%8zu  %4d  " INTPTR_FORMAT " " INTPTR_FORMAT " %s",
```

Comment:
Sounds like there are more than 8 digits now?

Reply:
I thought about this, too, but I don't think it's a problem because the width is specified like this: "Minimum number of characters to be printed. If the value to be printed is shorter than this number, the result is padded with blank spaces. The value is not truncated even if the result is larger." [https://cplusplus.com/reference/cstdio/printf/].
Do we want a larger fixed number of digits?

Reply:
Yeah, it is not about the correctness. It is more about readability: if we expect more than 8 digits, then the "table" we are printing here would be a bit ragged. UINT64_MAX is about 20 digits. In practice we would probably never do this for longer than 1 hour, and with (ballparking) 100M/sec bytecodes, this gives us a practical upper limit of 12 digits or so? My math might be off a digit or two.

Reply:
Actually, nevermind. I don't think this is useful to adjust. The bytecode counter is global, so it is not a per-bytecode printout like I initially thought.